conversation/chat-app/src/routes/+page.svelte

```diff
@@ -119,8 +121,9 @@
       </div>
       <div class="m-12 w-2/5">
         <script
-          src="https://www.gstatic.com/dialogflow-console/fast/messenger-cx/prod/df-messenger.min.js?v=1"></script>
+          src="https://www.gstatic.com/dialogflow-console/fast/df-messenger/prod/v1/df-messenger.js"></script>
         <df-messenger
+          project-id="koverholt-devrel-355716"
```

Comment:
Is there a way you can pass this in a config from the backend instead of hardcoding it in the repo?
Although, it does look like the published website has this value in the source code too 😅

(At least replace with a placeholder here so it's not published)

Reply:
Good point, I should be consistent with the values in https://github.com/GoogleCloudPlatform/generative-ai/blob/main/conversation/chat-app/.firebaserc and https://github.com/GoogleCloudPlatform/generative-ai/blob/main/conversation/chat-app/firebase.json. Fixed in bdd3388.